tests/Unit/Parallel/Test_AlgorithmCore.cpp

```diff
@@ -518,7 +518,7 @@ struct finalize {
     SPECTRE_PARALLEL_REQUIRE(db::get<TemporalId>(box) ==
                              TestAlgorithmArrayInstance{4});
     SPECTRE_PARALLEL_REQUIRE(db::get<CountActionsCalled>(box) == 13);
-    SPECTRE_PARALLEL_REQUIRE(db::get<Int1>(box) == 5);
+    SPECTRE_PARALLEL_REQUIRE(db::get<Int1>(box) == 10);
```

Comment:
why did this change?

Reply:
This tag counts the number of times that `is_ready` is called for this component. My understanding is that the count can be different because the `execute_next_phase` calls both `start_phase` and `receive_data`, so the count depends on the ordering of those messages. Previously, the test actually varied what the count will be if it is run in parallel, but was consistently 5 in serial.
Now, it is consistently 10 (always running an `is_ready` check that returns false before the receive_data), regardless of how many processors the test is run on. My understanding is that this is because the `perform_algorithm` call is also inline now, so in consistently calls the `execute_next_phase` operations in order.